stories/core/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { SvgPlaceholder, SvgSettings } from '@itwin/itwinui-icons-react';
+import { useState } from '@storybook/addons';
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  SideNavigation,
+  SidenavButton,
+  SideNavigationProps,
+} from '../../src/core';
+
+export default {
+  component: SideNavigation,
+  subcomponents: { SidenavButton },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    mainItems: { control: { disable: true } },
+    secondaryItems: { control: { disable: true } },
+  },
+  args: { style: { height: 'calc(100vh - 24px) ' } },
+  title: 'Core/SideNavigation',
+} as Meta<SideNavigationProps>;
+
+export const Basic: Story<SideNavigationProps> = (args) => {
+  return (
+    <SideNavigation
+      {...args}
+      mainItems={[
+        <SidenavButton startIcon={<SvgPlaceholder />} key={0}>
```

Comment:
For me placeholder svgs look ugly here, can we have some different svg like home? But maybe it is only for me 😀

Reply:
Home, Flag and maybe something from Folder?

Reply:
Done
![image](https://user-images.githubusercontent.com/9084735/118992926-11fab800-b953-11eb-8693-5dc73c44d4b3.png)